composer.json

```diff
@@ -2,18 +2,23 @@
   "name": "silverstripe/graphql-devtools",
   "description": "Tools to help developers building new applications on SilverStripe’s GraphQL API",
   "type": "silverstripe-vendormodule",
+  "license": "BSD-3-Clause",
   "require": {
-    "silverstripe/graphql": "*"
+    "silverstripe/graphql": "^3 || ^4",
+    "symfony/finder": "^4 || ^5",
+    "symfony/filesystem": "^4 || ^5"
+    },
+  "require-dev": {
+    "phpunit/phpunit": "^9.5",
+    "squizlabs/php_codesniffer": "^3.0"
   },
   "autoload": {
     "psr-4": {
-      "SilverStripe\\GraphQLDevTools\\": "src/"
+      "SilverStripe\\GraphQLDevTools\\": "src/",
+      "SilverStripe\\GraphQLDevTools\\Tests\\": "tests/"
     }
   },
   "extra": {
-    "branch-alias": {
-	  "dev-master": "1.x-dev"
```

Comment:
Why is this being removed?

Reply:
There a history of these things being in Silverstripe modules, they're generally more annoying that helpful. These days we remove these things any time we see them. I don't think this needs a separate PR.

Reply:
Oh there's no `1` branch and no stable tags.  I guess if someone out there is requiring this as ^1 then that'll break.

Reply:
> I don't think this needs a separate PR

I agree it's okay in this PR so long as we can say why it's happening, and if it _should_ happen. Which you've now provided! Thanks.
In this case I'd say it's okay. We're not even tagging this module, so anyone with `^1` as a constraint with this module imo has the wrong constraint, and it _should_ break for them, as a sanity check.

Reply:
My instinct would be to start tagging release numbers for this.